spec/javascripts/fixtures/json/gis-api-valid-address-boundary-match.json

```diff
@@ -1 +1 @@
-{"gis_data":{"address":"1222 HARRISON ST, San Francisco, CA","location":{"x":-13626517.2840026,"y":4547486.54839139},"score":100,"attributes":{"Loc_name":"eas"},"extent":{"xmin":-13626717.2840026,"ymin":4547286.54839139,"xmax":-13626317.2840026,"ymax":4547686.54839139},"boundary_match":true}}
```

Comment:
did float precision change in ruby 2.5 or something?

Reply:
Seems like it: https://nithinbekal.com/posts/ruby-2-4-features/